pkg/alertmanager/amcfg.go

```diff
@@ -309,13 +309,7 @@ func (cb *configBuilder) addAlertmanagerConfigs(ctx context.Context, amConfigs m
 	// amConfigIdentifiers is a sorted slice of keys from
 	// amConfigs map, used to always generate the config in the
 	// same order.
-	amConfigIdentifiers := make([]string, len(amConfigs))
-	i := 0
-	for k := range amConfigs {
-		amConfigIdentifiers[i] = k
-		i++
-	}
-	sort.Strings(amConfigIdentifiers)
+	amConfigIdentifiers := util.SortedKeys(amConfigs)
 
 	subRoutes := make([]*route, 0, len(amConfigs))
 	for _, amConfigIdentifier := range amConfigIdentifiers {
```

Comment:
we could get rid of the intermediate `amConfigIdentifiers` variable?

Reply:
Thanks @simonpasquier .
Remove the intermediate variable done.